tdvt/tdvt/config/tdvt/tdvt.ini

```diff
@@ -1,3 +1,3 @@
 [DEFAULT]
-TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau 2019.1/bin/tabquerytool.exe
-TAB_CLI_EXE_MAC = /Applications/Tableau 2019.1.app/Contents/MacOs/tabquerytool
+TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau 2019.2/bin/tabquerytool.exe
+TAB_CLI_EXE_MAC = /Applications/Tableau 2019.2.app/Contents/MacOs/tabquerytool
```

Comment:
```suggestion
TAB_CLI_EXE_MAC = /Applications/Tableau 2019.2.app/Contents/MacOS/tabquerytool
```
Should be `MacOS` not `MacOs`. Check out dir in bottom of below screenshot.

<img width="1121" alt="Screen Shot 2019-08-20 at 8 43 08 AM" src="https://user-images.githubusercontent.com/5643640/63362367-90fa3100-c326-11e9-8ff0-de1c3bff47a1.png">

`/Applications/Tableau Desktop 2019.2.app/Contents/MacOS/tabquerytool`

Reply:
Changed it.